charts/posthog/values.yaml

```diff
@@ -1337,6 +1338,12 @@ pgbouncer:
   ## -- PgBouncer pod(s) annotation.
   podAnnotations: {}
 
+pgbouncer:
+  <<: *_pgbouncer
+
+pgbouncer_read:
```

Comment:
Maybe for consistency:

```suggestion
pgbouncerRead:
```

Although I don't know what the generally used format is in the community.

Reply:
Ahh good point, will change it